packages/raygui/raygui.0.3.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for raygui"
+description: "OCaml bindings for raygui"
+maintainer: ["tobiasjammer@gmail.com"]
+authors: ["Tobias Mock"]
+license: "MIT"
+homepage: "https://github.com/tjammer/raylib-ocaml"
+bug-reports: "https://github.com/tjammer/raylib-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "dune-configurator"
+  "ctypes" {>= "0.14"}
+  "ppx_cstubs" {>= "0.6"}
+  "raylib" {>= "0.3"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/tjammer/raylib-ocaml.git"
+available: [arch != "arm32" & arch != "ppc64"]
+x-commit-hash: "d04c2066294c95fab7d5fe2f1f795502e988fd92"
+x-ci-accept-failures: [
+  "centos-7"
+  "oraclelinux-7"
+]
```

Comment:
```suggestion
```
This is not needed.Only raylib will fail. CI will detect this.

Reply:
Raygui also builds some c stubs, so the build of raygui will fail on these distros for the same reason, independant of raylib

Reply:
Ok fair enough. Could you add the same comment than in raylib though (just so that it is easier to maintain/understand why this is here in the future)